src/librustc_driver/lib.rs

```diff
@@ -1059,6 +1066,12 @@ pub fn handle_options(args: &[String]) -> Option<getopts::Matches> {
         return None;
     }
 
+    // Late check to see if @file was used without unstable options enabled
+    if crate::args::used_unstable_argsfile() && !nightly_options::is_unstable_enabled(&matches) {
```

Comment:
I guess this should go up to where the other check_nightly_options is (or maybe in check_nightly_options?).

Reply:
Fixed